### DISCUSSION TITLE
breaking up large projects vs `shortcut`

### DISCUSSION THREAD

**Discussion by mkoohafkan** [ropensci/targets#763] [Help]
I have a use case where I am computing literally hundreds of thousands of targets (static --> dynamic branching) but batching is not a great solution for my case. I am trying to understand if I should split my analysis into multiple target projects/scripts or if I can instead use the `shortcut` argument to `tar_make()` to run my analysis in chunks (which would be similar to how I would choose to manually split my workflow into separate projects). Will using `shortcut` reduce the potential overhead from `tar_make()`, or am I better off breaking my analysis into separate projects?

**Comment by wlandau** [comment]
It depends on how well the components of your project separate into different sub-projects and how fast `tar_make(shortcut = TRUE)` runs for the pipeline you have now. 

With hundreds of thousands of targets, it may be worth considering a simplification strategy that goes further than batching, e.g. removing one of the levels of branching altogether in favor of putting all that looping inside a single target.
